src/instrumentations/amqplib/AmqplibInstrumentation.test.js

```diff
@@ -0,0 +1,22 @@
+import LumigoAmqplibInstrumentation from './AmqplibInstrumentation';
+
+describe('LumigoAmqplibInstrumentation', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  let lumigoAmqplibInstrumentation = new LumigoAmqplibInstrumentation();
+
+  test('getInstrumentedModule should return "amqplib"', () => {
+    expect(lumigoAmqplibInstrumentation.getInstrumentedModule()).toEqual('amqplib');
+  });
+
+  test.skip('requireIfAvailable should return required name', () => {
```

Comment:
why did you add skipped test? 🤔

Reply:
i didn't intend to...

Reply:
i take that back, it wasn't a mistake, it's a copy paste from the other instrumentation tests. i'm going to unskip it and see what happens

Reply:
it's not clear why these tests were skipped, so i'm unskipping the lot of them (incl. for grpc)

Reply:
it seems like they're taking tons of time, and I'm not sure what is their impact 🤷  please consult with Moses about it :)

Reply:
as discussed out of band, i'm considering this out of scope and have created https://lumigo.atlassian.net/browse/RD-11195